### DISCUSSION TITLE
CRUD reliability across different DB versions.

### DISCUSSION THREAD

**Discussion by gk-patel** [prisma/prisma#16527] [Q&A] [answered]
Hi Prisa Community, 

I am new to Prisma, but really impressed with the node-client and the community-vibes. 
I am planning to write a multi-tenant (or multi-db) app. Which means my app, should be able to connect with Postgress hosted by different entities who also host the app (whos code is managed centrally, but not deployment). 
So at a given time, the app should be able to connect to different versions of the database which is running at the respective entity. 
Unfortunately, due to large number of combinations, it is not possible to test the app against all possible DB versions. 
Can we assume that prisma models and queries would deliver similar results across different db version for Postgress ? 

Thank for your time and support.

**Comment by nurul3101** [comment, marked as answer]
Hello @gk-patel 👋 

We have an [ecosystem-tests](https://github.com/prisma/ecosystem-tests) setup which tests PrismaClient with different operating systems, databases, frameworks, platforms and other setups. And these are the [databases and versions](https://www.prisma.io/docs/reference/database-reference/supported-databases) which we support, so you shouldn't face any issues in these versions.

**Reply by vladi-stevanovic** [reply]
👋 Hi @gk-patel , did you perhaps have a chance to check Nurul's answer? Let us know if you still have any questions! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
